kuma/static/styles/components/home/home-masthead.scss

```diff
@@ -5,12 +5,20 @@ homepage masthead (search, feature callouts)
 .home-masthead {
     @include clearfix();
     position: relative;
+    background-color: $accent-light;
```

Comment:
In the color meter, it appears the old background was ``#82CCEB`` text on ``#88D1F1``. The new background is on ``#7ECBED`` on ``#83D0F2``, at least with the color meter. Is this why the dino head appears purple, while the old one appeared grey?

Reply:
No, I reckon the dino head just needs a bit of a colour adjustments.